crates/shared/src/arguments.rs

```diff
@@ -435,13 +436,20 @@ impl FromStr for ExternalSolver {
     type Err = anyhow::Error;
 
     fn from_str(solver: &str) -> Result<Self> {
-        let (name, url) = solver
-            .split_once('|')
-            .context("not enough arguments for external solver")?;
+        let parts: Vec<&str> = solver.split('|').collect();
```

Comment:
Bit unfortunate for 2 reasons:
1. more weird custom formats
2. this is in shared which means it can also be configured in the orderbook despite it not doing anything there

But I think unless we switch to config files for orderbook and autopilot this is the most reasonable option.